### PR TITLE
chore: move utils to ts

### PR DESCRIPTION
**Pull request by dni**
part 5

### CODE REVIEW

File: src/utils/feeChecker.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
-const relevantMinerFees = (fees) => {
+const relevantMinerFees = (fees: any) => {
     return asset() === BTC
         ? fees.minerFees.quoteAsset
         : fees.minerFees.baseAsset;
 };
 
-export const feeChecker = (pairs) => {
+export const feeChecker = (pairs: any) => {
```
Comment (michael1011) [inline src/utils/feeChecker.ts:10]: Here we should use strong types. The format of the data is known

File: src/utils/denomination.ts [hunk 2]
```diff
@@ -43,16 +50,20 @@ export const formatAmountDenomination = (denom, amount, fixed = false) => {
...
-    switch (denomination()) {
+export const convertAmount = (amount: number, denom: string): number => {
+    switch (denom) {
         case denominations.btc:
             const amountBig = new BigNumber(amount).multipliedBy(satFactor);
             return amountBig.toNumber();
         case denominations.sat:
             return Number(amount);
+        default:
+            return Number(amount);
```
Comment (michael1011) [inline src/utils/denomination.ts:66]:
```suggestion
        default:
            return Number(amount);
```

File: src/utils/denomination.ts [hunk 3]
```diff
@@ -43,16 +50,20 @@ export const formatAmountDenomination = (denom, amount, fixed = false) => {
 
         case denominations.sat:
             return amount.toString();
+        default:
+            return amount.toString();
```
Comment (michael1011) [inline src/utils/denomination.ts:54]:
```suggestion
        default:
            return amount.toString();
```

File: tests/components/Fees.spec.tsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ import { cfg } from "../config";
 
 describe("Fees component", () => {
     beforeAll(() => {
+        // @ts-ignore
```
Comment (michael1011) [inline tests/components/Fees.spec.tsx:11]: The type of `setConfig` is wrong. Don't just ts-ignore here. That defeats the purpose of typescript

**Review by michael1011 — CHANGES_REQUESTED** [review]

File: src/utils/feeChecker.ts (hunk 1 above)
Comment (dni) [reply src/utils/feeChecker.ts:10]: open issue for that

**Pull request merged**